docs/NLUGitHubActions.md

```diff
@@ -0,0 +1,85 @@
+## GitHub Actions workflow using NLU.DevOps cli tool
+
+This documents walk through building a CI pipeline for LUIS using GitHub Actions and NLU.DevOps.
+The workflow will be the following: 
+1. create, train, publish the LUIS model using sample utterances
+2. send a test set to created LUIS model
+3. evaluate model by comparing results received from LUIS with expected values
+4. delete the model from the portal
+
+Let's name this pipeline CINLU and run it on a pull request to GitHub repo.
+
+```
+name: CINLU
+
+on: [pull_request]
+```
+
+1. Let's install NLU.DevOps cli tool on GitHub agent.
```

Comment:
It's okay to just be imperative, you can probably remove `Let's install` and just say `Install`